README.md

```diff
@@ -29,46 +29,73 @@ Our package fills a gap in the python ecosystem by being marketed specifically t
 
 ## Installation
 
+### For User
+
+To install the package, run the command pip install quanteda
+
+### For Developers
+
 1. Clone this repository.
 
 ```bash
-$ git clone git@github.com:UBC-MDS/quanteda.git
-$ cd quanteda/
+ git clone git@github.com:UBC-MDS/quanteda.git
+ cd quanteda/
 ``` 
 
 2. Install the virtual environment.
 
 ```bash
-$ conda env create -f environment.yml
+ conda env create -f environment.yml
 ```
 
 3. Activate the installed environment:
 
 ```bash
-$ conda activate quanteda
+ conda activate quanteda
 ```    
 
 4. Install the package.
 
 ```bash
-$ poetry install
+ poetry install
 ```
-
 ## Usage
 
+### Using this package
+To use this package, import and call the functions in Python. Below is an example:
+
+```bash
```

Comment:
I think this would help the code's color be more python-like
```suggestion
```python
```

Reply:
Great suggestions! I updated it accordingly. Thanks very much!